deck.rb

```diff
@@ -0,0 +1,35 @@
+# The deck should let us remove a card. It should tell us if a card is valid.
+# After removing a card, the card should no longer be valid.
+require './card'
+
+class Deck
+
+	def initialize
+		@cards = []
+		@suits = ["H" , "D", "S", "C"]
+		@values = [1,2,3]
+	
+		@suits.each do |suit| 
+			@values.each do |value|
+				@cards.push(Card.new(suit, value))
+			end
+		end 
+	end
+
+	def validate(card)
```

Comment:
In Ruby, the convention with methods that return boolean values is to name them as if they are questions you can ask the class. For example, `deck.valid_card?(card)` would make it immediately clear what the function validates (a card) and what it will return (a boolean representing whether or not the card is valid). Here's a good thread about the convention: https://stackoverflow.com/questions/37059547/naming-conventions-for-boolean-attributes